securing_apps/topics/saml/mod-auth-mellon.adoc

```diff
@@ -91,6 +91,25 @@ To configure the Mellon service provider, complete the following steps:
 
 NOTE: Some of the files referenced in the code above are created in later steps.
 
+==== Setting the SameSite value for the cookie used by mod_auth_mellon
+
+Browsers are planning to set the default value for the `SameSite` attribute for cookies to `Lax`. This setting means
+that cookies will be sent to applications only if the request originates in the same domain. This behavior can affect
+the SAML POST binding which may become non-functional. To preserve full functionality of the SAML adapter, we recommend
```

Comment:
> To preserve full functionality of the SAML adapter

I don't think this is SAML adapter, right? :)

Reply:
Of course. Thanks!